src/main/resources/edu/cornell/kfs/module/ld/businessobject/datadictionary/PositionData.xml

```diff
@@ -57,45 +57,32 @@
   </bean>
       
   <!-- Business Object Inquiry Definition -->
+  <bean id="PositionData-inquiryDefinition-override" parent="DataDictionaryBeanOverride">
+    <property name="beanName" value="PositionData-inquiryDefinition" />
+    <property name="fieldOverrides">
+      <list>
+          <!-- Bean override for new standalone page field presentation -->
+          <bean parent="FieldOverrideForListElementInsert"
+            p:propertyName="sections[0].fields"
+            p:propertyNameForElementCompare="name"
+            p:element-ref="PositionData-departmentId"
+            p:insertAfter-ref="PositionData-extension-orgCode"/>
 
-  <bean parent="DataDictionaryBeanOverride">
-      <property name="beanName" value="PositionData-inquiryDefinition" />
-      <property name="fieldOverrides">
-          <list>
-              <bean parent="FieldOverrideForListElementInsert">
-                  <property name="propertyName" value="inquirySections[0].inquiryFields" />
-                  <property name="propertyNameForElementCompare" value="attributeName" />
-                  <property name="element">
-                      <bean parent="FieldDefinition" p:attributeName="departmentId" />
-                  </property>
-                  <property name="insertAfter">
-                      <list>
-                          <bean parent="FieldDefinition" p:attributeName="extension.orgCode" />
-                      </list>
-                  </property>
-              </bean>
-          </list>
-      </property>
-  </bean>
-  
-  <bean parent="DataDictionaryBeanOverride">
-      <property name="beanName" value="PositionData-inquiryDefinition" />
-      <property name="fieldOverrides">
-          <list>
-              <bean parent="FieldOverrideForListElementInsert">
-                  <property name="propertyName" value="inquirySections[0].inquiryFields" />
-                  <property name="propertyNameForElementCompare" value="attributeName" />
-                  <property name="element">
-                      <bean parent="FieldDefinition" p:attributeName="extension.inactivationDate" />
-                  </property>
-                  <property name="insertAfter">
-                      <list>
-                          <bean parent="FieldDefinition" p:attributeName="extension.inactivationDate" />
```

Comment:
Please add the "extension.inactivationDate" insertion back in accordingly, along with an equivalent insertion for the new inquiry setup. I see that this setup is confusing since the field to insert is also the field to match on, but upon further analysis of the bean's code, it looks like the "insertAfter" element will be placed at the end of the list if the element to match on is not defined or is not present. You could try omitting the "element" property for this particular case, if you can verify that the application still starts up properly and the inquiry still places the field at the end of the list.